ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -13,13 +13,28 @@ function (
         if (!initialised) {
             initialised = true;
 
-            window.injectCreative = injectCreative;
-            window.applyNativeFunctionCall('injectInlineCreative');
+            if (GU.opts.contentType === 'liveblog' || document.querySelector('.article__body--liveblog')) {
+              trackLiveBlogEpic();
+            } else {
+              window.injectCreative = injectCreative;
+              window.applyNativeFunctionCall('injectInlineCreative');
+            }
         }
     }
 
+    function trackLiveBlogEpic() {
+      var liveBlogEpicContainer = document.querySelector('contributions-epic');
+      var liveBlogEpicContainerId;
+
+      if (liveBlogEpicContainer) {
+          liveBlogEpicContainerId = liveBlogEpicContainer.getAttribute('id');
+      }
+
+      window.addEventListener('scroll', util.debounce(isCreativeInView.bind(null, liveBlogEpicContainer, liveBlogEpicContainerId), 100));
```

Comment:
The addEventListener should go inside the if liveBlogEpicContainer block incase no liveBlogEpicContainer/liveBlogEpicContainerId exists. 

Could we also pull out the entire window.addEventListener line and put it inside a function? There's an addEventListener on line 71 we could then replace with a function call as it's exactly the same as this line?